.github/workflows/pr_created.yaml

```diff
@@ -0,0 +1,36 @@
+name: PR Created
+
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+    branches:
+      - master
+
+jobs:
+  post-pr-data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Post PR data
```

Comment:
We should turn this into a GitHub action which can be shared.
Eg. `- uses: open-chat-labs/github-bot@v1`

Reply:
Yes - I'll look into how to do that.